src/pages/auth/register.tsx

```diff
@@ -34,17 +39,55 @@ const Register = () => {
     setEmailInvalid(false);
     setEmail(e.target.value);
   };
+  const handleToast = (label: string, type?: TostifyType) => {
+    UseTostify({ label, type });
+  };
```

Comment:
```suggestion
  const handleToast = (label: string, type?: TostifyType) => UseTostify({ label, type });
```

Reply:
i removed "{}" but eslint still putting in two different lines.